clients/privacy-center/cypress/e2e/consent-banner.cy.ts

```diff
@@ -683,6 +692,9 @@ describe("Consent banner", () => {
             expect(cookieKeyConsent.consent)
               .property(PRIVACY_NOTICE_KEY_1)
               .is.eql(false);
+            expect(cookieKeyConsent.fides_meta)
+              .property("consentMethod")
+              .is.eql(ConsentMethod.GPC);
```

Comment:
I added these kinds of assertions to existing tests. Hopefully pretty good coverage 👍